geth/jail/console/console_test.go

```diff
@@ -0,0 +1,97 @@
+package console_test
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/robertkrimen/otto"
+	"github.com/status-im/status-go/geth/jail/console"
+	"github.com/status-im/status-go/geth/node"
+	"github.com/stretchr/testify/suite"
+)
+
+// TestConsole validates the behaviour of giving conole extensions.
+func TestConsole(t *testing.T) {
+	suite.Run(t, new(ConsoleTestSuite))
+}
+
+type ConsoleTestSuite struct {
+	suite.Suite
+	vm *otto.Otto
+}
+
+func (s *ConsoleTestSuite) SetupTest() {
+	vm := otto.New()
```

Comment:
This is super minor but I think it's beneficial to remember about it, not only in Go. If it's possible to keep a variable declaration and its usage close, let's do that. Here, we could group it like:
```go
require := s.Require()

vm := otto.New()
require.NotNil(vm)
require.IsType(&otto.Otto{}, vm) // btw. do you think it's still necessary if vm is get from otto.New()?
s.vm = vm
```